manifests/charts/istio-cni/values.yaml

```diff
@@ -32,6 +32,12 @@ defaults:
     # Allows user to set custom affinity for the DaemonSet
     affinity: {}
 
+    # Custom labels on Deployment level, if you need them
```

Comment:
This isn't a Deployment so the naming is kind of weird?

Reply:
That was a typo. Fixed it